### ISSUE TITLE
Iframe resize parameter causes syntax issue if no banner available for the invoked zone

### ISSUE THREAD

**Issue by jeerbl** [revive-adserver/revive-adserver#1028]
Problem:
When using iframes, when setting the `resize` parameter to 1 but no banner is allowed in the zone (let's say because every banner linked to that zone is filtered by delivery options), the function `MAX_adjust_frame` does not exist because `$banner['width']` and `$banner['height']` are empty, causing a syntax error.

Error is (JavaScript console):
```
Uncaught SyntaxError: Unexpected token ;
Uncaught ReferenceError: MAX_adjustframe is not defined
    at onload (afr.php?resize=1&zoneid=8&target=_blank&cb=54617&issuer=nobannerhere:23)
```

The source code of the MAX_adjust_frame function in the iframe is:
```javascript
function MAX_adjustframe(frame) {
    if (document.all) {
        parent.document.all[frame.name].width = ;
        parent.document.all[frame.name].height = ;
    }
    else if (document.getElementById) {
        parent.document.getElementById(frame.name).width = ;
        parent.document.getElementById(frame.name).height = ;
    }
}
```

Steps To Reproduce:
Create a banner in a zone and invoke the zone with an iframe. Set the `resize` parameter to 1 in the invocation code. Set the delivery option of that banner that will cause the banner to be filtered. The zone has to have only this banner linked to it.

**Comment by jeerbl** [comment]
@mbeccati when can I expect a new version with the fix? 4.1.5 I'm assuming

Thanks!
Jerome

**Comment by andrewatfornax** [comment]
Hi @jeerbl,

The fix should be in the next release, whatever version that happens to be (depending on what we include in it). Our higher level Patreon supporters get advance notice of releases! :-)
